src/dmn/trn_transit_xdp_usr.c

```diff
@@ -129,7 +130,7 @@ int trn_bpf_maps_init(struct user_metadata_t *md)
 	md->ing_vsip_except_map = bpf_map__next(md->ing_vsip_supp_map, md->obj);
 	md->conn_track_cache = bpf_map__next(md->ing_vsip_except_map, md->obj);
 
-	if (!md->networks_map || !md->vpc_map || !md->endpoints_map ||
+	if (!md->networks_map || !md->vpc_map || !md->endpoints_map || !md->packet_metadata_map ||
```

Comment:
Can this check be added after conn_track_cache?

Reply:
Fixed.